src/tests/ftest/server/multiengine_persocket.py

```diff
@@ -196,28 +195,32 @@ def test_multiengines_per_socket(self):
         :avocado: tags=server
         :avocado: tags=test_multiengines_per_socket
         """
-        # (1) Storage prepare --scm-ns-per-socket
+        # (1) Scm reset and prepare --scm-ns-per-socket
         step = 1
-        self.log.info("===(%s)===Storage prepare --scm-ns-per-socket", step)
-        self.daos_server_storage_prepare_reset(step)
+        self.log.info("===(%s)===Scm reset and prepare --scm-ns-per-socket", step)
         engines_per_socket = self.params.get(
             "engines_per_socket", "/run/server_config/*", default=1)
-        self.daos_server_storage_prepare_ns(step, engines_per_socket)
+        self.daos_server_scm_reset(step)
+        self.host_reboot(self.hostlist_servers)
```

Comment:
This method shutdown the node sleep for 300 seconds and assumes the node is booted. There is no check to verify whether the node is really active before proceeding to next command. This can cause issue... Maybe, think about it for future enhancement in the script. Since this is a manual test case, it is fine for now. For large scale testing, it will be nice to verify the nodes status.